qemu/tests/win_virtio_driver_installer_repair.py

```diff
@@ -99,6 +99,11 @@ def run(test, params, env):
             driver_test_params = params.get('driver_test_params_%s'
                                             % driver_name, '{}')
             driver_test_params = ast.literal_eval(driver_test_params)
+            if driver_name == "viofs":
+                win_driver_installer_test.run_viofs_service(test, params,
+                                                            session)
+            if driver_name != "balloon":
+                driver_test_params = ast.literal_eval(driver_test_params)
```

Comment:
hi @nickzhq , I am curious about why we run ast.literal_eval twice when driver_name is viofs, do we hit error as driver_test_params_viofs is not set? Did I miss anything?

Reply:
> hi @nickzhq , I am curious about why we run ast.literal_eval twice when driver_name is viofs, do we hit error as driver_test_params_viofs is not set? Did I miss anything?

Hi @zhencliu , the reason is code conflict during "git rebase", there is no worry at here. However, this mistake has been solved in third commit at this patch.